Week7/CatApi/app/src/main/java/com/example/catapi/networking/RemoteApi.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.catapi.networking
+
+import com.example.catapi.model.*
+
```

Comment:
Nice work connecting the api! I know it's difficult to get lists, you may want to explore TypeConverter for lists within the data.